### PR TITLE
Runs on Windows

### PR DESCRIPTION
**Pull request by Claytonhaddock**
Added instruction to get running in CMD

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -16,9 +16,11 @@
 If you get it working on Windows, you should make an instruction file and
 submit a pull request!
 
-1. Clone or unzip this repository into your chosen directory
-2. Open a terminal to said directory and type `npm install` (this will create a *large* `node_modules` directory)
-3. Type `gulp` - the app should open in your default browser
++1. Clone or unzip this repository into your chosen directory
++2. Open a terminal to said directory and type `npm install` (this will create a *large* `node_modules` directory)
++2.5. (Windows only) In the terminal window type 'npm install -g gulp'. This allows the CLI to access gulp
```
Comment (jpetto) [inline Readme.md]: Actually, this is also a requirement for OS X (I just didn't specify). Could you add gulp (linking to the [npm package](https://www.npmjs.com/package/gulp)) to the requirements bullets instead?

File: Readme.md [hunk 2]
```diff
@@ -16,9 +16,11 @@
...
++3. Add the URL to your firebase account "var base = Rebase.createClass('YOUR URL HERE');" in ListManager.js
++4. Type `gulp` - the app should open in your default browser
```
Comment (jpetto) [inline Readme.md]: No need for the extra `+` before each number. Please remote those.

File: Readme.md [hunk 3]
```diff
@@ -16,9 +16,11 @@
 If you get it working on Windows, you should make an instruction file and
 submit a pull request!
 
-1. Clone or unzip this repository into your chosen directory
-2. Open a terminal to said directory and type `npm install` (this will create a *large* `node_modules` directory)
-3. Type `gulp` - the app should open in your default browser
++1. Clone or unzip this repository into your chosen directory
++2. Open a terminal to said directory and type `npm install` (this will create a *large* `node_modules` directory)
++2.5. (Windows only) In the terminal window type 'npm install -g gulp'. This allows the CLI to access gulp
++3. Add the URL to your firebase account "var base = Rebase.createClass('YOUR URL HERE');" in ListManager.js
```
Comment (jpetto) [inline Readme.md]: Two things here:
1. Wrap the code (`var base = ...`) in backticks for nice formatting. (Underneath the tilde, left of the 1 key (on a Mac, anyway).)
2. For the uninitiated, specify the full path to `ListManager.js`: `js/components/ListManager.js`

**Comment by jpetto** [comment]
This is a great PR @Claytonhaddock! Would like a couple things cleaned up and then will merge.

Thanks!

File: Readme.md (hunk 1 above)
Comment (Claytonhaddock) [inline Readme.md]: After moving gulp to requirements should I remove step "2.5" entirely? Or just remove the "(Windows only)" part?

Comment (jpetto) [inline Readme.md]: Yep, I'd get rid of the 2.5 step entirely. :+1:

**Pull request closed without merging**